upstream: no
reason: don't prompt for password, don't assume it has been asked for

--- tests/cert-tests/certtool
+++ tests/cert-tests/certtool
@@ -26,6 +26,7 @@
 TMPFILE1=certtool-file1.$$.tmp
 TMPFILE2=certtool-file2.$$.tmp
 PASS="1234"
+export GNUTLS_PIN="1234"
 
 if test -n "$DISABLE_BASH_TESTS"; then
 	exit 77
@@ -66,7 +67,7 @@
 	fi
 
 	grep "Enter password" ${TMPFILE2} >/dev/null 2>&1
-	if test $? != 0;then
+	if test $? != 0 && test -z "$GNUTLS_PIN" ; then
 		cat ${TMPFILE2}
 		echo "No password was asked"
 		exit 1
